activestorage/app/models/active_storage/blob.rb

```diff
@@ -183,7 +183,7 @@ def audio?
 
   # Returns true if the content_type of this blob is in the video range, like video/mp4.
   def video?
-    content_type.start_with?("video")
+    content_type.start_with?("video") || content_type == "application/mp4"
```

Comment:
For some reason `application/mp4` is returned instead of `video/mp4`, even though the created mp4 contains a video stream. Any idea why?

Reply:
Is it a marcel bug?

Reply:
This line apparently returns `application/mp4` for the variation but `video/mp4` for the original blob:
https://github.com/rails/rails/blob/d6a1cff4e477b8426118c319aa9dc8fa760680ed/activestorage/app/models/active_storage/blob.rb#L324 

So seems like a Marcel bug? The ffmpeg transformation I used in the test keeps the audio and video streams (they are just played twice slower). So Marcel should return `video/mp4`.

Reply:
I've tested the FfmpegTransformation and this now seems to be returning "video/mp4" as the content type, can you re-test it in your local environment?

Reply:
@guilleiguaran I've rebased, removed the workaround in blob#video (`|| content_type == "application/mp4"`), and the marcel bug doesn't seem to be there any more: the new tests pass (`cd activestorage && bin/test test/models/variant_with_record_test.rb`).